src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* max-width: 1200px;
+  min-width: 1024px; */
+  padding: 0 40px 0 50px;
```

```suggestion
  padding: 0 50px;
```
Падінги з обох боків мають бути по 50 пікселів. А от у останнього пункту меню просто не повинно бути відступу справа